website/settings/defaults.py

```diff
@@ -334,6 +334,13 @@ def parent_dir(path):
 ECSARXIV_CROSSREF_USERNAME = None
 ECSARXIV_CROSSREF_PASSWORD = None
 
+# if our DOIs cannot be comfirmed after X amount of days email the admin
+DAYS_CROSSREF_DOIS_MUST_BE_STUCK_BEFORE_EMAIL = 5
```

Comment:
Minor: I believe we say that it may take up to 24 hours for Crossref to issue a DOI; `2` might be a more reasonable value here.